integration-tests/api/__tests__/admin/order-edit.js

```diff
@@ -430,6 +430,44 @@ describe("[MEDUSA_FF_ORDER_EDITING] /admin/order-edits", () => {
     })
   })
 
+  describe("POST /admin/order-edits/:id/request-confirmation", () => {
+    let orderEditId
+    beforeEach(async () => {
+      await adminSeeder(dbConnection)
+
+      const { id } = await simpleOrderEditFactory(dbConnection, {
+        created_by: "admin_user",
+      })
+
+      orderEditId = id
+    })
+
+    afterEach(async () => {
+      const db = useDb()
+      return await db.teardown()
+    })
+
+    it("requests order edit", async () => {
```

Comment:
question: should we throw an error if we request confirmation without changes? it is most likely an error in that case?